added reacting to poll

featrd3 opened the pull request:
Added recognition of reactions to poll:
Added function for handling binary voting (uses thumbsup and thumbsdown emotes as means of voting for and against), mainly as means of acccepting input displayed in embed message. Other reactions are ignored in voting process.

featrd3 committed:
added reacting to poll

patrycjabru commented on BotCommands/LoLCommands.cs:
```diff
@@ -26,10 +31,19 @@ public async Task HelpLol(CommandContext ctx)
             var embed = LoLCommandsSupp.LolEmbedBuilder(gameData);
 
-            await ctx.RespondAsync(embed: embed.Build()).ConfigureAwait(false);
+            var duration = TimeSpan.FromSeconds(10);
```
You can put this "10 seconds" in the configuration file.

patrycjabru commented on BotCommandsSupplement/CommonBotCommands.cs:
```diff
@@ -0,0 +1,54 @@
+                var creatorGaveThumbsDown = thumbsDownReaction.Users.Where(x => x.Username.Equals(ctx.User.Username)).FirstOrDefault();
+                if (creatorGaveThumbsDown != null)
+                {
+                    return (false);
```
Those brackets are not needed here.

patrycjabru commented on DiscordBot/Bot.cs:
```diff
@@ -27,16 +30,23 @@ public async Task RunAsync()
             Client = new DiscordClient(config);
             Client.Ready += OnClientReady;
 
+            Client.UseInteractivity(new InteractivityConfiguration()
```
This can be encapsulated in some well named method for example AddEmojiInteractivity() or something like this.

featrd3 committed:
Added poll duration and interactivity duration as value in config file, changed reading of config to use already implemented functions, fixed remarks from pull request

featrd3 commented:
Changed accordingly:
- moved poll duration to config file
- removed brackets from "CommonBotCommands"

Did not encapsulate "UseInteractivity", to keep it consistent with other similar bot settings in the same class.

Also fixed loading data from config file in "Bot" class to use already present functionality.

featrd3 committed:
Merge branch 'main' into AddingPolltoLOLCommand
